### ISSUE TITLE
JS error on sce editor

### ISSUE THREAD

**Issue by albertlast** [SimpleMachines/SMF#4685]
Not sure if this bug is in smf or in sce:
![grafik](https://user-images.githubusercontent.com/1782906/38171229-aefeccbc-3595-11e8-97dd-0619e9d124e0.png)
Is shown in the console when you got quick replay enabled or go in the replay.
For the screenshot i enabled pretty printer in ff.

@Sesquipedalian maybe you could look into

**Comment by Sesquipedalian** [comment]
I can't reproduce this in Safari, Firefox, or Chrome.

**Comment by frandominguezl** [comment]
I can't reproduce it either.

**Comment by XinYenFon** [comment]
Disable editor via admin panel and try again I can reproduce the error.

**Comment by frandominguezl** [comment]
I can reproduce it doing what Antes said and using FireFox. Not with Chrome.

**Comment by Sesquipedalian** [comment]
I was also able to reproduce it using those instructions. I tested using the development version as well, and found that it produced the same error on line 6205 of jquery.sceditor.bbcode.js:
```
TypeError: wysiwygWindow.getSelection(...) is null
```

I found that the errors stopped if I changed line 6205 from this:
```
if (wysiwygWindow.getSelection().rangeCount <= 0) {
```
... to this:
```
if (wysiwygWindow.getSelection() !== null && wysiwygWindow.getSelection().rangeCount <= 0) {
```

This suggests to me that the bug is in SCEditor rather than in anything we are doing. Specifically, it seems that this line in SCEditor's code assumes that `wysiwygWindow.getSelection()` will never be null, when in fact it can be.

We should probably create an issue on https://github.com/samclarke/SCEditor. In the meantime, I don't think we need to worry about this. As far as I can tell, the error does not break any functionality.
